frontend/cli.py

```diff
@@ -361,22 +373,23 @@ def wells(bbox, county,
     Get locations
     """
 
-    config = setup_config("sites", bbox, county, site_limit, dry)
+    config = setup_config("sites", config, bbox, county, site_limit, dry)
```

Comment:
see comment about preserving the variable `config` for a `Config` object. Perhaps use `config_yml_file` or something similar to keep the code easier to read/follow